pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -24,48 +24,58 @@ let
     Google’s answer to tofu. The name noto is to convey the idea that
     Google’s goal is to see “no more tofu”.  Noto has multiple styles and
     weights, and freely available to all.
-
-    This package also includes the Arimo, Cousine, and Tinos fonts.
   '';
 in
 rec {
   mkNoto =
     { pname
-    , weights
     , variants ? [ ]
     , longDescription ? notoLongDescription
     }:
     stdenvNoCC.mkDerivation rec {
       inherit pname;
-      version = "20201206-phase3";
+      version = "unstable-2023-01-17";
 
       src = fetchFromGitHub {
-        owner = "googlefonts";
-        repo = "noto-fonts";
-        rev = "v${version}";
-        hash = "sha256-x60RvCRFLoGe0CNvswROnDkIsUFbWH+/laN8q2qkUPk=";
+        owner = "notofonts";
+        repo = "notofonts.github.io";
+        rev = "e14f32b06b6f53538f171c7ba9d4ad4ab368afe0";
+        hash = "sha256-geUIVjAq6CvAI5rnBYbtLMRgVAW+LY2knYi3cuc6J3Q=";
       };
 
       _variants = map (variant: builtins.replaceStrings [ " " ] [ "" ] variant) variants;
 
       installPhase = ''
-        # We copy in reverse preference order -- unhinted first, then
-        # hinted -- to get the "best" version of each font while
+        # We check avaibility in order of variable -> otf -> ttf
+        # unhinted -- the hinted versions use autohint
         # maintaining maximum coverage.
-        #
-        # TODO: install OpenType, variable versions?
-        local out_ttf=$out/share/fonts/truetype/noto
+
+        # We have a mix of otf and ttf fonts
+        local out_font=$out/share/fonts/noto
+
+        # Remove duplicate font
+        rm -rf fonts/NotoLoopedThai
```

Comment:
What is it a duplicate of?

Reply:
Noto Sans Thai Looped got removed in https://github.com/notofonts/notofonts.github.io/commit/5adbed9e370be27ad684faf7df912f822a113ffb but got re-added in https://github.com/notofonts/notofonts.github.io/commit/453578f49c4ea06cc074a73c710fef3273349b44